wiki/heath_potion.md

```diff
@@ -0,0 +1,26 @@
+Turrets 
+=======
+
+* Icon: `♥`
+* Buy with: `l, 1, *`
+* Use with: `u, 1, *`
+* Cost: `50 Ore`
+
+#### Services Offered
+
+Health Potions give the user an instant boost of 15 health directly to their health bar.
```

Comment:
To their health bar? That kinda sounds weird. Say this instead

`Health Potions give their user an instant health boost of 15.`